spec/model/post_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  # let(:user){User.new(name: 'Kay', posts_counter: 1)}
```

Comment:
- [ Required ] I think it would be better if you remove this commented code from your code base since it makes your project look unprofessional. Commented code is also considered to be a sign of code that is not clean, kindly delete any piece of commented code in your code base.